org.contourterminal.Contour.yml

```diff
@@ -0,0 +1,106 @@
+app-id: org.contourterminal.Contour
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
+  - --device=all          # we need access to at least DRI (for OpenGL) & PTY
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak # Required for process `flatpak-spawn --host ...`.
+
+modules:
+  - name: contour
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCONTOUR_INSTALL_TOOLS=ON"
+      - "-DCONTOUR_BLUR_PLATFORM_KWIN=ON"
+      - "-DCONTOUR_TESTING=OFF"
+      - "-DLIBTERMINAL_TESTING=OFF"
+    # NOTE: This post-install should be removed with the next release of Contour, when metainfo.xml is auto-installed.
+    post-install:
+      - install -p -m755 -d /app/share/metainfo
+      - install -p -m644 org.contourterminal.Contour.metainfo.xml /app/share/metainfo
+      - install -p -m644 org.contourterminal.Contour.desktop /app/share/applications
+      - rm /app/share/applications/contour.desktop
```

Comment:
we have `rename-desktop-file` for that, You should use this file and not include one here.

Reply:
I used `rename-desktop-file` initially, but then I realized that `flatpak-builder` locally is generating a warning during desktop file validation stage, so I fixed it by providing a fixed one and having the right name in place right away. I could insist in using rename-desktop-file (as used in a previous commit) but then apply a patch. But this solution was more clean to me.